Sources/Plasma/FeatureLib/pfMetalPipeline/ShaderSrc/GammaCorrection.metal

```diff
@@ -79,3 +79,17 @@ fragment half4 gammaCorrectFragment(GammaVertexOut in               [[stage_in]]
         1.0
     };
 }
+
+#if defined(__METAL_IOS__) || __METAL_VERSION__ >= 230
+fragment half4 gammaCorrectFragmentInPlace(GammaVertexOut in        [[stage_in]],
```

Comment:
Where does this function actually get used? I see the binding to `gammaCorrectFragment` but not conditionally to `gammaCorrectFragmentInPlace`

Reply:
There's a follow up PR to actually use the function. This PR was busy enough I didn't want to bring it in. I did want to have an Metal 2.3 only function present though to test that optional Metal 2.3 paths were actually working.